client/src/templates/Challenges/classic/lower-jaw.tsx

```diff
@@ -150,6 +169,8 @@ const LowerJaw = ({
   const [isFeedbackHidden, setIsFeedbackHidden] = useState(false);
   const { t } = useTranslation();
   const testFeedbackRef = React.createRef<HTMLDivElement>();
+  const endOfProject = React.useRef<boolean>(false);
+  const isBlockCompleted = 100;
```

Comment:
```suggestion
  const { id: superBlock, block } = challengeMeta;
```

Reply:
I am destructing it, when taking props itself into function. so i think it what you suggested for destructing and my code destructing looks almost similar. 

Still if you want me to improve i can improve it

<img width="524" alt="image" src="https://user-images.githubusercontent.com/36238382/230704653-80952243-c4ee-492b-ab41-837c04e63334.png">